KnockKnock-iOS/Scenes/Challenge/ChallengeViewController.swift

```diff
@@ -56,7 +56,7 @@ final class ChallengeViewController: BaseViewController<ChallengeView> {
 }
 
 extension ChallengeViewController: ChallengeViewProtocol {
-  func fetchChallenges(challenges: [Challenge]) {
+  func fetchChallenges(challenges: [Challenges]) {
```

Comment:
```suggestion
  func fetchChallenges(challenges: [Challenge]) {
```
Challenges 가 맞을까요?? Challenges 가 적절해 보이긴 하는데 말이죠 🤔

Reply:
api에서 challenges로 표기하셔서 싱크 맞추어 사용했습니다!